service/src/main/java/cz/muni/fi/service/DriverServiceImpl.java

```diff
@@ -1,4 +1,106 @@
 package cz.muni.fi.service;
 
+import cz.muni.fi.dao.DriverDao;
+import cz.muni.fi.entities.Driver;
+import java.util.List;
+import org.dozer.inject.Inject;
+import cz.muni.fi.service.exception.TeamFormulaDataAccessException;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Lucie Kureckova, 445264
+ */
+@Service
 public class DriverServiceImpl implements DriverService {
-}
+
+    @Inject
+    private DriverDao driverDao;
+    
+    @Override
+    public List<Driver> listDrivers() {
+        try{
+            return driverDao.listDrivers();
+        } catch(Throwable ex){
+            throw new FormulaTeamDataAccessException(ex.getMessage()) {};
```

Comment:
throw ServiceDataAccessException instead

Reply:
Sorry i forgot, please specify the message, f.e. in list `ServiceDataException("Cannot list all drivers", ex);` thanks